addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2514,6 +2514,55 @@ QUnit.test('chat window should remain folded when new message is received', asyn
     );
 });
 
+QUnit.test('Resizing the composer should not modify the scroll bar position', async function(assert) {
```

Comment:
Would formulate test name in a clearer way, e.g. _"with conversation scrolled to end, when composer size changes, conversation should still be scrolled to end"_.

It's a bit longer and quite new way to name test, but I think it's better to understand the intent of test when it's written like a user story.